tutorials/hist/Normalizing_Histogram.C

```diff
@@ -0,0 +1,171 @@
+/// \file
+/// \ingroup tutorial_hist
+/// \notebook -js
+/// Normalizing a Histogram
```

Comment:
Maybe nicer: "Normalize a Histogram" / `NormalizeHistogram.C`?

Reply:
Yeah, that sounds better